source/hassio/installation.markdown

```diff
@@ -152,12 +152,18 @@ apt-get install -y apparmor-utils apt-transport-https avahi-daemon ca-certificat
 curl -fsSL get.docker.com | sh
 ```
 
-And to install Hass.io the one below. That one is used also for other distributions.
+The below script works for generic installs on x86/x64 machines, running an OS such as Ubuntu. 
```

Comment:
I think the original one was better, especially this is not true. 
Hass.io is not limited to x86/x64 machines. Please restore the original line.